### DISCUSSION TITLE
关于如何解决标签泄露问题

### DISCUSSION THREAD

**Discussion by Echo628999** [pykt-team/pykt-toolkit#139] [Q&A]

**Comment by Patrick-bj** [comment]
Hi,感谢您对我们工作的支持，您目前图片上的test结果是one-by-one的结果，会涉及到标签泄露问题，我们提供了fusion预测缓解知识点信息泄露问题(pykt-toolkit/pykt/models/evaluate_model.py line 360).

**Comment by Patrick-bj** [comment]
fustion预测(pykt-toolkit/pykt/examples/wandb_predict.py line 78/86)，您结果：testauc/window_testauc分别对应一一非window和window的结果（即KC级别），auclate_xx/windowauclate_xx是我们question级别的结果，也就是进行了fusion后的结果

**Reply by Echo628999** [reply]
还想向您咨询一下ori_ _中ori是什么意思，另外oriaucconcepts中concepts对应的是哪种预测场景

**Comment by Patrick-bj** [comment]
非window直接截断，大于最大长度x的部分会做为全新的学生序列，大于x的部分构成的新序列无法重新利用历史信息。 window不截断，会使用最近的x-1个历史进行下一时刻的预测。ori对应的非window模式，关于concept，我们的文章中没有介绍这个方法 ---- 回复的原邮件 ***@***.***>发送日期2023年11月06日 21:27 ***@***.***> ***@***.***>***@***.***>主题Re: [pykt-team/pykt-toolkit] 关于如何解决标签泄露问题 (Discussion #139)
还想向您咨询一下ori_ _中ori是什么意思，另外oriaucconcepts中concepts对应的是哪种预测场景

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/pykt-team/pykt-toolkit/discussions/139#discussioncomment-7487534",
"url": "https://github.com/pykt-team/pykt-toolkit/discussions/139#discussioncomment-7487534",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
